Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN \
     apt-get install -y nodejs; \
     cd kinto/plugins/admin; npm install; npm run build; \
     pip3 install -e /app[postgresql,memcached,monitoring] -c /app/requirements.txt; \
-    pip3 install kinto-pusher kinto-attachment ; \
+    pip3 install kinto-attachment kinto-emailer kinto-elasticsearch kinto-portier kinto-redis; \
```

Comment:
Are those official plugins that we want to ship with Kinto? Why would we want to install them all by default? Should we plan to move them to kinto.plugins?

Reply:
I took the ones that were not coupled to any brand or commercial service